client/webserver/site/src/js/locales.ts

```diff
@@ -123,7 +125,9 @@ export const enUS: Locale = {
   [ID_WALLET_ENABLED_MSG]: '{{ assetName }} Wallet Enabled',
   [ID_WALLET_DISABLED_MSG]: '{{ assetName }} Wallet Disabled',
   [ID_DISABLED_MSG]: 'wallet is disabled',
-  [ID_ACTIVE_ORDERS_ERR_MSG]: '{{ assetName }} wallet is actively managing orders'
+  [ID_ACTIVE_ORDERS_ERR_MSG]: '{{ assetName }} wallet is actively managing orders',
+  [ID_INVALID_ADDRESS_MSG]: 'Invalid address: {{ address }}',
+  [ID_TXFEE_UNAVAILABLE]: 'Fee estimation for this {{ assetName }} transaction is unavailable at the moment. This could be due to an error while calculating transaction fees or wallet does not currently support fee estimation.'
```

Comment:
This PR can add some brief messages on top of the changes in https://github.com/decred/dcrdex/pull/1870, but I want to go ahead with what I have there that considers the trait and skips the /txfee request entirely when it is know the wallet does not support that.

Also, with that trait check in mind, the handling changes and we don't need a long message like this that alludes to different possible reasons:
- if trait not detected -> form just says "fee estimation is not supported for this wallet type"
- if trait detected but an error is returned from /api/txfee -> show the actual error, "fee estimation failed: <reason>"

Reply:
> EDIT: you can build on the traits check in https://github.com/decred/dcrdex/commit/1dbeb380fd3d95e53c85143ccf7888b02d44bd46, which I've removed from https://github.com/decred/dcrdex/pull/1870

Okay.